src/main/scala/beam/router/r5/RoutingWorker.scala

```diff
@@ -1,47 +1,60 @@
 package beam.router.r5
 
+import java.io.File
```

Comment:
Shall this moved to `beam.router` namespace?

Reply:
done